common/group.cpp

```diff
@@ -94,6 +94,8 @@ std::ostream& operator<<(std::ostream& ss, fty::Group::Fields value)
                 return "hosted-by";
             case fty::Group::Fields::Group:
                 return "group";
+            case fty::Group::Fields::Tag:
+                return "tags";
```

Comment:
must be 'tag' (no plural), as other tokens...

Reply:
'tags' - specified in documentation, enum 'Tag' - to be plural